group_vars/donbot_azure/vars.yml

```diff
@@ -24,9 +24,9 @@ local_admin_users:
   - 'morris'
   - 'pieter'
   - 'sandi'
+use_sssd: true
 use_ldap: false
```

Comment:
Two questions:
1.  Why is it also specified in the static inventory? There is currently only one server in this stack, so the group_vars should work for all machines in the stack and not require an override for a specific machine yet.
2. Why is use_ldap false here (but true in the static_inventory)? AFAIK this combination is not supported. When
- `use_ldap` is `false` then both the old `ldap` role and the newer `sssd` role will not be used.
-  `use_ldap` is `true` and `use_sssd` is either false or not specified, then the older `ldap` role is used.
-  `use_ldap` is `true` and `use_sssd` is also `true` then the `sssd` role is used and the old `ldap` role is not used.
This logic can be simplified as we do not support nor use the old `ldap` role anymore. So the `ldap` role as well as the `use_ldap` variable can be deleted. Either we use local accounts or we use accounts from an LDAP server via SSSD.